Libraries/LibJS/AST.h

```diff
@@ -387,6 +418,10 @@ class CallExpression : public Expression {
 
 enum class AssignmentOp {
     Assign,
+    PlusEquals,
+    MinusEquals,
+    AsteriskEquals,
```

Comment:
Personal opinion: I think these should be named similar to the following.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators

Reply:
Yep, I'd agree with that.